### DISCUSSION TITLE
Orchestrator function GetDnsZones_Orchestrator failed.

### DISCUSSION THREAD

**Discussion by charlesprakash** [polymind-inc/acmebot#260] [Q&A] [answered]
**Describe the bug**
After deploying the solution, when the URL /add-certificate is launched, the UI shows a dropdown and the immediately disappears.

Upon investigating the logs, the following message is found: `Orchestrator function 'GetDnsZones_Orchestrator' failed: The activity function 'GetZones' failed: "Specified method is not supported."`.

**To Reproduce**
1. Follow the getting started guide and setup the AcmeBot
2. Complete configuration with Azure AD auth
3. Launch the `<function-url>/add-certificate`

**Comment by shibayan** [comment, marked as answer]
You will need to follow the README below to set up your DNS provider.
https://github.com/shibayan/keyvault-acmebot#4-add-settings-for-your-choice-dns-provider

**Reply by charlesprakash** [reply]
Thanks a lot @shibayan, just missed the config entry, but did everything else. :-). And also, thanks for this wonderful tool.

**Reply by charlesprakash** [reply]
Also the same issue as: #253
